INFO:__main__:Get urls from stories
  0%|          | 0/30 [00:00<?, ?it/s]  3%|▎         | 1/30 [00:00<00:04,  6.76it/s]  7%|▋         | 2/30 [00:00<00:06,  4.09it/s] 10%|█         | 3/30 [00:01<00:10,  2.46it/s] 17%|█▋        | 5/30 [00:02<00:10,  2.39it/s] 23%|██▎       | 7/30 [00:03<00:10,  2.23it/s] 30%|███       | 9/30 [00:03<00:07,  2.74it/s] 40%|████      | 12/30 [00:04<00:05,  3.03it/s] 43%|████▎     | 13/30 [00:05<00:09,  1.88it/s] 60%|██████    | 18/30 [00:06<00:05,  2.32it/s] 67%|██████▋   | 20/30 [00:07<00:04,  2.34it/s] 77%|███████▋  | 23/30 [00:07<00:02,  3.15it/s] 80%|████████  | 24/30 [00:08<00:02,  2.48it/s] 83%|████████▎ | 25/30 [00:08<00:02,  2.33it/s] 93%|█████████▎| 28/30 [00:08<00:00,  3.20it/s]100%|██████████| 30/30 [00:09<00:00,  2.94it/s]
  0%|          | 0/157 [00:00<?, ?it/s]INFO:__main__:Requesting https://medium.com/@kloudtrader/private-beta-and-kloudtrader-aural-c4a8561f70d0
INFO:__main__:Getting metadata for https://medium.com/@kloudtrader/private-beta-and-kloudtrader-aural-c4a8561f70d0
  1%|          | 1/157 [00:00<01:10,  2.22it/s]INFO:__main__:Requesting https://kloudtrader.com
INFO:__main__:Getting metadata for https://www.kloudtrader.com
  1%|▏         | 2/157 [00:01<01:42,  1.51it/s]INFO:__main__:Requesting https://hackaday.com/2019/03/09/speedrunning-windows-95/
ERROR:__main__:Could not reach https://hackaday.com/2019/03/09/speedrunning-windows-95/
Traceback (most recent call last):
  File "/home/travis/virtualenv/python3.6.3/lib/python3.6/site-packages/urllib3/connectionpool.py", line 600, in urlopen
    chunked=chunked)
  File "/home/travis/virtualenv/python3.6.3/lib/python3.6/site-packages/urllib3/connectionpool.py", line 384, in _make_request
    six.raise_from(e, None)
  File "<string>", line 2, in raise_from
  File "/home/travis/virtualenv/python3.6.3/lib/python3.6/site-packages/urllib3/connectionpool.py", line 380, in _make_request
    httplib_response = conn.getresponse()
  File "/opt/python/3.6.3/lib/python3.6/http/client.py", line 1331, in getresponse
    response.begin()
  File "/opt/python/3.6.3/lib/python3.6/http/client.py", line 297, in begin
    version, status, reason = self._read_status()
  File "/opt/python/3.6.3/lib/python3.6/http/client.py", line 266, in _read_status
    raise RemoteDisconnected("Remote end closed connection without"
http.client.RemoteDisconnected: Remote end closed connection without response

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/travis/virtualenv/python3.6.3/lib/python3.6/site-packages/requests/adapters.py", line 449, in send
    timeout=timeout
  File "/home/travis/virtualenv/python3.6.3/lib/python3.6/site-packages/urllib3/connectionpool.py", line 638, in urlopen
    _stacktrace=sys.exc_info()[2])
  File "/home/travis/virtualenv/python3.6.3/lib/python3.6/site-packages/urllib3/util/retry.py", line 367, in increment
    raise six.reraise(type(error), error, _stacktrace)
  File "/home/travis/virtualenv/python3.6.3/lib/python3.6/site-packages/urllib3/packages/six.py", line 685, in reraise
    raise value.with_traceback(tb)
  File "/home/travis/virtualenv/python3.6.3/lib/python3.6/site-packages/urllib3/connectionpool.py", line 600, in urlopen
    chunked=chunked)
  File "/home/travis/virtualenv/python3.6.3/lib/python3.6/site-packages/urllib3/connectionpool.py", line 384, in _make_request
    six.raise_from(e, None)
  File "<string>", line 2, in raise_from
  File "/home/travis/virtualenv/python3.6.3/lib/python3.6/site-packages/urllib3/connectionpool.py", line 380, in _make_request
    httplib_response = conn.getresponse()
  File "/opt/python/3.6.3/lib/python3.6/http/client.py", line 1331, in getresponse
    response.begin()
  File "/opt/python/3.6.3/lib/python3.6/http/client.py", line 297, in begin
    version, status, reason = self._read_status()
  File "/opt/python/3.6.3/lib/python3.6/http/client.py", line 266, in _read_status
    raise RemoteDisconnected("Remote end closed connection without"
urllib3.exceptions.ProtocolError: ('Connection aborted.', RemoteDisconnected('Remote end closed connection without response',))

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "../_scripts/crawler/crawler.py", line 210, in metafy_url
    response = requests.get(url, timeout=6, headers={'User-Agent': 'Mozilla/5.0'})
  File "/home/travis/virtualenv/python3.6.3/lib/python3.6/site-packages/requests/api.py", line 75, in get
    return request('get', url, params=params, **kwargs)
  File "/home/travis/virtualenv/python3.6.3/lib/python3.6/site-packages/requests/api.py", line 60, in request
    return session.request(method=method, url=url, **kwargs)
  File "/home/travis/virtualenv/python3.6.3/lib/python3.6/site-packages/requests/sessions.py", line 533, in request
    resp = self.send(prep, **send_kwargs)
  File "/home/travis/virtualenv/python3.6.3/lib/python3.6/site-packages/requests/sessions.py", line 646, in send
    r = adapter.send(request, **kwargs)
  File "/home/travis/virtualenv/python3.6.3/lib/python3.6/site-packages/requests/adapters.py", line 498, in send
    raise ConnectionError(err, request=request)
requests.exceptions.ConnectionError: ('Connection aborted.', RemoteDisconnected('Remote end closed connection without response',))
  2%|▏         | 3/157 [00:01<01:19,  1.94it/s]INFO:__main__:Requesting https://msfn.org/board/topic/141402-windows-95-21ghz-cpu-limit-broken/
INFO:__main__:Getting metadata for https://msfn.org/board/topic/141402-windows-95-21ghz-cpu-limit-broken/
  3%|▎         | 4/157 [00:03<02:01,  1.26it/s]INFO:__main__:Requesting https://mobile.twitter.com/Foone/status/1103384684812255233
INFO:__main__:Getting metadata for https://mobile.twitter.com/Foone/status/1103384684812255233
  3%|▎         | 5/157 [00:05<02:45,  1.09s/it]INFO:__main__:Requesting https://en.wikipedia.org/wiki/NetBIOS_Frames_protocol
INFO:__main__:Getting metadata for https://en.wikipedia.org/wiki/NetBIOS_Frames_protocol
  4%|▍         | 6/157 [00:05<02:31,  1.00s/it]INFO:__main__:Requesting https://news.vice.com/en_us/article/7xakd9/windows-31-is-still-alive-and-it-just-killed-a-french-airport
INFO:__main__:Getting metadata for https://news.vice.com/en_us/article/7xakd9/windows-31-is-still-alive-and-it-just-killed-a-french-airport
  4%|▍         | 7/157 [00:06<01:54,  1.31it/s]INFO:__main__:Requesting https://arstechnica.com/information-technology/2017/05/in-a-throwback-to-the-90s-ntfs-bug-lets-anyone-hang-or-crash-windows-7-8-1/
INFO:__main__:Getting metadata for https://arstechnica.com/information-technology/2017/05/in-a-throwback-to-the-90s-ntfs-bug-lets-anyone-hang-or-crash-windows-7-8-1/
  5%|▌         | 8/157 [00:06<01:31,  1.64it/s]INFO:__main__:Requesting https://xkcd.com/284/
INFO:__main__:Getting metadata for https://xkcd.com/284/
  6%|▌         | 9/157 [00:06<01:21,  1.83it/s]INFO:__main__:Requesting http://www.annoyances.org/
INFO:__main__:Getting metadata for http://www.annoyances.org
  6%|▋         | 10/157 [00:06<01:05,  2.26it/s]INFO:__main__:Requesting https://www.litepc.com/
INFO:__main__:Getting metadata for https://www.litepc.com
  7%|▋         | 11/157 [00:10<03:42,  1.53s/it]INFO:__main__:Requesting https://github.com/CypherpunkArmory/UserLAnd
INFO:__main__:Getting metadata for https://github.com/CypherpunkArmory/UserLAnd
  8%|▊         | 12/157 [00:11<03:13,  1.33s/it]INFO:__main__:Requesting https://wiki.termux.com/wiki/Main_Page
INFO:__main__:Getting metadata for https://wiki.termux.com/wiki/Main_Page
  8%|▊         | 13/157 [00:15<04:44,  1.97s/it]INFO:__main__:Requesting https://github.com/termux/termux-app
INFO:__main__:Getting metadata for https://github.com/termux/termux-app
  9%|▉         | 14/157 [00:16<03:56,  1.65s/it]INFO:__main__:Requesting https://www.reddit.com/r/termux/
INFO:__main__:Getting metadata for https://www.reddit.com/r/termux/
 10%|▉         | 15/157 [00:17<03:54,  1.65s/it]INFO:__main__:Requesting https://github.com/termux/termux-app/issues/954#issuecomment-448199494
INFO:__main__:Getting metadata for https://github.com/termux/termux-app/issues/954#issuecomment-448199494
 10%|█         | 16/157 [00:19<03:35,  1.53s/it]INFO:__main__:Requesting https://github.com/sp4rkie/debian-on-termux
INFO:__main__:Getting metadata for https://github.com/sp4rkie/debian-on-termux
 11%|█         | 17/157 [00:19<02:59,  1.28s/it]INFO:__main__:Requesting https://github.com/its-pointless/gcc_termux
INFO:__main__:Getting metadata for https://github.com/its-pointless/gcc_termux
 11%|█▏        | 18/157 [00:20<02:35,  1.12s/it]INFO:__main__:Requesting https://play.google.com/store/apps/details?id=org.connectbot
INFO:__main__:Getting metadata for https://play.google.com/store/apps/details?id=org.connectbot
 12%|█▏        | 19/157 [00:21<02:13,  1.03it/s]INFO:__main__:Requesting https://play.google.com/store/apps/details?id=org.pocketworkstation.pckeyboard
INFO:__main__:Getting metadata for https://play.google.com/store/apps/details?id=org.pocketworkstation.pckeyboard
 13%|█▎        | 20/157 [00:21<01:55,  1.19it/s]INFO:__main__:Requesting https://browser.geekbench.com/android_devices/839
INFO:__main__:Getting metadata for https://browser.geekbench.com/android_devices/839
 13%|█▎        | 21/157 [00:27<05:05,  2.25s/it]INFO:__main__:Requesting https://browser.geekbench.com/processors/2294
INFO:__main__:Getting metadata for https://browser.geekbench.com/processors/2294
 14%|█▍        | 22/157 [00:32<07:08,  3.17s/it]INFO:__main__:Requesting https://browser.geekbench.com/processors/763
INFO:__main__:Getting metadata for https://browser.geekbench.com/processors/763
 15%|█▍        | 23/157 [00:35<07:16,  3.26s/it]INFO:__main__:Requesting https://www.samsung.com/semiconductor/insights/tech-trends/emmc-to-ufs-how-nand-memory-for-mobile-products-is-evolving/
INFO:__main__:Getting metadata for https://www.samsung.com/semiconductor/insights/tech-trends/emmc-to-ufs-how-nand-memory-for-mobile-products-is-evolving/
 15%|█▌        | 24/157 [00:36<05:29,  2.48s/it]INFO:__main__:Requesting https://www.microsoft.com/en-gb/windows/continuum
INFO:__main__:Getting metadata for https://www.microsoft.com/en-gb/windows/continuum
 16%|█▌        | 25/157 [00:37<04:24,  2.00s/it]INFO:__main__:Requesting https://maruos.com
INFO:__main__:Getting metadata for https://maruos.com
 17%|█▋        | 26/157 [00:38<03:43,  1.71s/it]INFO:__main__:Requesting https://ubports.com/
INFO:__main__:Getting metadata for https://ubports.com
 17%|█▋        | 27/157 [00:39<03:20,  1.55s/it]INFO:__main__:Requesting https://github.com/louisdh/openterm
INFO:__main__:Getting metadata for https://github.com/louisdh/openterm
 18%|█▊        | 28/157 [00:40<02:58,  1.39s/it]INFO:__main__:Requesting https://github.com/ColdGrub1384/LibTerm/blob/master/README.md
INFO:__main__:Getting metadata for https://github.com/ColdGrub1384/LibTerm/blob/master/README.md
 18%|█▊        | 29/157 [00:41<02:32,  1.20s/it]INFO:__main__:Requesting https://github.com/tbodt/ish
INFO:__main__:Getting metadata for https://github.com/tbodt/ish
 19%|█▉        | 30/157 [00:42<02:25,  1.15s/it]INFO:__main__:Requesting https://liliputing.com/2018/07/gpd-pocket-2-preview.html
INFO:__main__:Getting metadata for https://liliputing.com/2018/07/gpd-pocket-2-preview.html
 20%|█▉        | 31/157 [00:42<01:47,  1.17it/s]INFO:__main__:Requesting https://github.com/fsquillace/junest
INFO:__main__:Getting metadata for https://github.com/fsquillace/junest
 20%|██        | 32/157 [00:43<01:45,  1.19it/s]INFO:__main__:Requesting https://github.com/pirate/ArchiveBox
INFO:__main__:Getting metadata for https://github.com/pirate/ArchiveBox
 21%|██        | 33/157 [00:44<01:49,  1.13it/s]INFO:__main__:Requesting https://www.reuters.com/article/us-netherlands-pharmaceuticals-insight-idUSKCN1QP0M4
INFO:__main__:Getting metadata for https://www.reuters.com/article/us-netherlands-pharmaceuticals-insight-idUSKCN1QP0M4
 22%|██▏       | 34/157 [00:44<01:31,  1.34it/s]INFO:__main__:Requesting https://nos.nl/artikel/2266712-hoe-nederlands-kankeronderzoek-novartis-aan-miljarden-gaat-helpen.html
INFO:__main__:Getting metadata for https://nos.nl/artikel/2266712-hoe-nederlands-kankeronderzoek-novartis-aan-miljarden-gaat-helpen.html
 22%|██▏       | 35/157 [00:46<01:44,  1.17it/s]INFO:__main__:Requesting http://sourceonhealthcare.org/orphan-drug-act-fostering-innovation-or-abuse/
INFO:__main__:Getting metadata for http://sourceonhealthcare.org/orphan-drug-act-fostering-innovation-or-abuse/
 23%|██▎       | 36/157 [00:47<01:48,  1.12it/s]INFO:__main__:Requesting https://philanthropynewsdigest.org/news/cystic-fibrosis-foundation-receives-3.3-billion-royalty-pay-out
INFO:__main__:Getting metadata for https://philanthropynewsdigest.org/news/cystic-fibrosis-foundation-receives-3.3-billion-royalty-pay-out
 24%|██▎       | 37/157 [00:47<01:40,  1.20it/s]INFO:__main__:Requesting https://www.kff.org/news-summary/india-brazil-launch-trade-dispute-against-eu-netherlands-over-seizures-of-generic-drugs/
ERROR:__main__:Could not reach https://www.kff.org/news-summary/india-brazil-launch-trade-dispute-against-eu-netherlands-over-seizures-of-generic-drugs/
Traceback (most recent call last):
  File "/home/travis/virtualenv/python3.6.3/lib/python3.6/site-packages/urllib3/connectionpool.py", line 600, in urlopen
    chunked=chunked)
  File "/home/travis/virtualenv/python3.6.3/lib/python3.6/site-packages/urllib3/connectionpool.py", line 384, in _make_request
    six.raise_from(e, None)
  File "<string>", line 2, in raise_from
  File "/home/travis/virtualenv/python3.6.3/lib/python3.6/site-packages/urllib3/connectionpool.py", line 380, in _make_request
    httplib_response = conn.getresponse()
  File "/opt/python/3.6.3/lib/python3.6/http/client.py", line 1331, in getresponse
    response.begin()
  File "/opt/python/3.6.3/lib/python3.6/http/client.py", line 297, in begin
    version, status, reason = self._read_status()
  File "/opt/python/3.6.3/lib/python3.6/http/client.py", line 266, in _read_status
    raise RemoteDisconnected("Remote end closed connection without"
http.client.RemoteDisconnected: Remote end closed connection without response

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/travis/virtualenv/python3.6.3/lib/python3.6/site-packages/requests/adapters.py", line 449, in send
    timeout=timeout
  File "/home/travis/virtualenv/python3.6.3/lib/python3.6/site-packages/urllib3/connectionpool.py", line 638, in urlopen
    _stacktrace=sys.exc_info()[2])
  File "/home/travis/virtualenv/python3.6.3/lib/python3.6/site-packages/urllib3/util/retry.py", line 367, in increment
    raise six.reraise(type(error), error, _stacktrace)
  File "/home/travis/virtualenv/python3.6.3/lib/python3.6/site-packages/urllib3/packages/six.py", line 685, in reraise
    raise value.with_traceback(tb)
  File "/home/travis/virtualenv/python3.6.3/lib/python3.6/site-packages/urllib3/connectionpool.py", line 600, in urlopen
    chunked=chunked)
  File "/home/travis/virtualenv/python3.6.3/lib/python3.6/site-packages/urllib3/connectionpool.py", line 384, in _make_request
    six.raise_from(e, None)
  File "<string>", line 2, in raise_from
  File "/home/travis/virtualenv/python3.6.3/lib/python3.6/site-packages/urllib3/connectionpool.py", line 380, in _make_request
    httplib_response = conn.getresponse()
  File "/opt/python/3.6.3/lib/python3.6/http/client.py", line 1331, in getresponse
    response.begin()
  File "/opt/python/3.6.3/lib/python3.6/http/client.py", line 297, in begin
    version, status, reason = self._read_status()
  File "/opt/python/3.6.3/lib/python3.6/http/client.py", line 266, in _read_status
    raise RemoteDisconnected("Remote end closed connection without"
urllib3.exceptions.ProtocolError: ('Connection aborted.', RemoteDisconnected('Remote end closed connection without response',))

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "../_scripts/crawler/crawler.py", line 210, in metafy_url
    response = requests.get(url, timeout=6, headers={'User-Agent': 'Mozilla/5.0'})
  File "/home/travis/virtualenv/python3.6.3/lib/python3.6/site-packages/requests/api.py", line 75, in get
    return request('get', url, params=params, **kwargs)
  File "/home/travis/virtualenv/python3.6.3/lib/python3.6/site-packages/requests/api.py", line 60, in request
    return session.request(method=method, url=url, **kwargs)
  File "/home/travis/virtualenv/python3.6.3/lib/python3.6/site-packages/requests/sessions.py", line 533, in request
    resp = self.send(prep, **send_kwargs)
  File "/home/travis/virtualenv/python3.6.3/lib/python3.6/site-packages/requests/sessions.py", line 646, in send
    r = adapter.send(request, **kwargs)
  File "/home/travis/virtualenv/python3.6.3/lib/python3.6/site-packages/requests/adapters.py", line 498, in send
    raise ConnectionError(err, request=request)
requests.exceptions.ConnectionError: ('Connection aborted.', RemoteDisconnected('Remote end closed connection without response',))
 24%|██▍       | 38/157 [00:47<01:15,  1.57it/s]INFO:__main__:Requesting https://www.oncode.nl/research/programs
ERROR:__main__:Could not reach https://www.oncode.nl/research/programs
Traceback (most recent call last):
  File "/home/travis/virtualenv/python3.6.3/lib/python3.6/site-packages/urllib3/connectionpool.py", line 600, in urlopen
    chunked=chunked)
  File "/home/travis/virtualenv/python3.6.3/lib/python3.6/site-packages/urllib3/connectionpool.py", line 343, in _make_request
    self._validate_conn(conn)
  File "/home/travis/virtualenv/python3.6.3/lib/python3.6/site-packages/urllib3/connectionpool.py", line 839, in _validate_conn
    conn.connect()
  File "/home/travis/virtualenv/python3.6.3/lib/python3.6/site-packages/urllib3/connection.py", line 344, in connect
    ssl_context=context)
  File "/home/travis/virtualenv/python3.6.3/lib/python3.6/site-packages/urllib3/util/ssl_.py", line 344, in ssl_wrap_socket
    return context.wrap_socket(sock, server_hostname=server_hostname)
  File "/opt/python/3.6.3/lib/python3.6/ssl.py", line 407, in wrap_socket
    _context=self, _session=session)
  File "/opt/python/3.6.3/lib/python3.6/ssl.py", line 814, in __init__
    self.do_handshake()
  File "/opt/python/3.6.3/lib/python3.6/ssl.py", line 1068, in do_handshake
    self._sslobj.do_handshake()
  File "/opt/python/3.6.3/lib/python3.6/ssl.py", line 689, in do_handshake
    self._sslobj.do_handshake()
ssl.SSLError: [SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed (_ssl.c:777)

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/travis/virtualenv/python3.6.3/lib/python3.6/site-packages/requests/adapters.py", line 449, in send
    timeout=timeout
  File "/home/travis/virtualenv/python3.6.3/lib/python3.6/site-packages/urllib3/connectionpool.py", line 638, in urlopen
    _stacktrace=sys.exc_info()[2])
  File "/home/travis/virtualenv/python3.6.3/lib/python3.6/site-packages/urllib3/util/retry.py", line 398, in increment
    raise MaxRetryError(_pool, url, error or ResponseError(cause))
urllib3.exceptions.MaxRetryError: HTTPSConnectionPool(host='www.oncode.nl', port=443): Max retries exceeded with url: /research/programs (Caused by SSLError(SSLError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed (_ssl.c:777)'),))

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "../_scripts/crawler/crawler.py", line 210, in metafy_url
    response = requests.get(url, timeout=6, headers={'User-Agent': 'Mozilla/5.0'})
  File "/home/travis/virtualenv/python3.6.3/lib/python3.6/site-packages/requests/api.py", line 75, in get
    return request('get', url, params=params, **kwargs)
  File "/home/travis/virtualenv/python3.6.3/lib/python3.6/site-packages/requests/api.py", line 60, in request
    return session.request(method=method, url=url, **kwargs)
  File "/home/travis/virtualenv/python3.6.3/lib/python3.6/site-packages/requests/sessions.py", line 533, in request
    resp = self.send(prep, **send_kwargs)
  File "/home/travis/virtualenv/python3.6.3/lib/python3.6/site-packages/requests/sessions.py", line 646, in send
    r = adapter.send(request, **kwargs)
  File "/home/travis/virtualenv/python3.6.3/lib/python3.6/site-packages/requests/adapters.py", line 514, in send
    raise SSLError(e, request=request)
requests.exceptions.SSLError: HTTPSConnectionPool(host='www.oncode.nl', port=443): Max retries exceeded with url: /research/programs (Caused by SSLError(SSLError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed (_ssl.c:777)'),))
 25%|██▍       | 39/157 [00:48<01:05,  1.81it/s]INFO:__main__:Requesting https://en.wikipedia.org/wiki/ISDS
INFO:__main__:Getting metadata for https://en.wikipedia.org/wiki/ISDS
 25%|██▌       | 40/157 [00:49<01:36,  1.21it/s]INFO:__main__:Requesting http://www.squidi.net/three/index.php
 26%|██▌       | 41/157 [00:49<01:16,  1.51it/s]INFO:__main__:Requesting https://www.theatlantic.com/technology/archive/2019/03/china-has-stopped-accepting-our-trash/584131/
INFO:__main__:Getting metadata for https://www.theatlantic.com/technology/archive/2019/03/china-has-stopped-accepting-our-trash/584131/
 27%|██▋       | 42/157 [00:50<01:01,  1.88it/s]INFO:__main__:Requesting https://www.bath.ac.uk/case-studies/debunking-big-tobaccos-arguments-against-standardised-packaging/
INFO:__main__:Getting metadata for https://www.bath.ac.uk/case-studies/debunking-big-tobaccos-arguments-against-standardised-packaging/
 27%|██▋       | 43/157 [00:52<01:49,  1.05it/s]INFO:__main__:Requesting https://www.thisiseco.co.uk/news_and_blog/what-happens-to-waste-to-energy-incineration-ash.html
INFO:__main__:Getting metadata for https://www.thisiseco.co.uk/news_and_blog/what-happens-to-waste-to-energy-incineration-ash.html
 28%|██▊       | 44/157 [00:53<01:58,  1.05s/it]INFO:__main__:Requesting https://en.wikipedia.org/wiki/Green_Dot_(symbol)
INFO:__main__:Getting metadata for https://en.wikipedia.org/wiki/Green_Dot_(symbol)
 29%|██▊       | 45/157 [00:54<01:48,  1.03it/s]INFO:__main__:Requesting https://slate.com/business/2013/01/yacht-economics-rich-people-don-t-consume-very-much.html
INFO:__main__:Getting metadata for https://slate.com/business/2013/01/yacht-economics-rich-people-don-t-consume-very-much.html
 29%|██▉       | 46/157 [00:54<01:35,  1.16it/s]INFO:__main__:Requesting https://en.m.wikipedia.org/wiki/Plasma_gasification
INFO:__main__:Getting metadata for https://en.m.wikipedia.org/wiki/Plasma_gasification
 30%|██▉       | 47/157 [00:55<01:35,  1.15it/s]INFO:__main__:Requesting https://www.dailymotion.com/video/x6klo0p
INFO:__main__:Getting metadata for https://www.dailymotion.com/video/x6klo0p
 31%|███       | 48/157 [00:56<01:48,  1.01it/s]INFO:__main__:Requesting https://en.m.wikipedia.org/wiki/Mobro_4000
INFO:__main__:Getting metadata for https://en.m.wikipedia.org/wiki/Mobro_4000
 31%|███       | 49/157 [00:57<01:32,  1.16it/s]INFO:__main__:Requesting https://www.plasticchina.org/
INFO:__main__:Getting metadata for https://www.plasticchina.org
 32%|███▏      | 50/157 [01:01<03:10,  1.78s/it]INFO:__main__:Requesting https://purecycletech.com/
INFO:__main__:Getting metadata for https://purecycletech.com
 32%|███▏      | 51/157 [01:02<02:38,  1.50s/it]INFO:__main__:Requesting https://pacman128.github.io/pcasm/
INFO:__main__:Getting metadata for https://pacman128.github.io/pcasm/
 33%|███▎      | 52/157 [01:02<02:02,  1.16s/it]INFO:__main__:Requesting http://www.yosefk.com/blog/its-done-in-hardware-so-its-cheap.html
INFO:__main__:Getting metadata for http://yosefk.com/blog/its-done-in-hardware-so-its-cheap.html
 34%|███▍      | 53/157 [01:04<02:23,  1.38s/it]INFO:__main__:Requesting https://github.com/pjc50/ancient-3d-for-turboc
INFO:__main__:Getting metadata for https://github.com/pjc50/ancient-3d-for-turboc
 34%|███▍      | 54/157 [01:05<02:02,  1.18s/it]INFO:__main__:Requesting https://en.wikipedia.org/wiki/3dfx_Interactive
INFO:__main__:Getting metadata for https://en.wikipedia.org/wiki/3dfx_Interactive
 35%|███▌      | 55/157 [01:06<02:02,  1.20s/it]INFO:__main__:Requesting https://en.wikipedia.org/wiki/Silicon_Graphics
INFO:__main__:Getting metadata for https://en.wikipedia.org/wiki/Silicon_Graphics
 36%|███▌      | 56/157 [01:08<02:11,  1.30s/it]INFO:__main__:Requesting https://en.wikipedia.org/wiki/Cache_on_a_stick
INFO:__main__:Getting metadata for https://en.wikipedia.org/wiki/Cache_on_a_stick
 36%|███▋      | 57/157 [01:08<01:52,  1.13s/it]INFO:__main__:Requesting https://latinamericareports.com/the-demolition-of-escobar-monaco-in-medellins-historical-memory
INFO:__main__:Getting metadata for https://latinamericareports.com/the-demolition-of-escobar-monaco-in-medellins-historical-memory/1294/
 37%|███▋      | 58/157 [01:10<02:17,  1.39s/it]INFO:__main__:Requesting http://www.10zenmonkeys.com/2019/02/25/the-great-captain-marvel-soundtrack-scam/
INFO:__main__:Getting metadata for http://www.10zenmonkeys.com/2019/02/25/the-great-captain-marvel-soundtrack-scam/
 38%|███▊      | 59/157 [01:13<03:09,  1.94s/it]INFO:__main__:Requesting https://mathml.igalia.com/news/2019/02/12/launch-of-the-project/
INFO:__main__:Getting metadata for https://mathml.igalia.com/news/2019/02/12/launch-of-the-project/
 38%|███▊      | 60/157 [01:16<03:20,  2.07s/it]INFO:__main__:Requesting https://en.wikipedia.org/wiki/MathML#Example_and_comparison_to_other_formats
INFO:__main__:Getting metadata for https://en.wikipedia.org/wiki/MathML#Example_and_comparison_to_other_formats
 39%|███▉      | 61/157 [01:18<03:06,  1.94s/it]INFO:__main__:Requesting https://gist.github.com/aplaice/266b092bc48afbbdd46cdbd0ca811da8#file-quadratic-xml
INFO:__main__:Getting metadata for https://gist.github.com/aplaice/266b092bc48afbbdd46cdbd0ca811da8#file-quadratic-xml
 39%|███▉      | 62/157 [01:18<02:25,  1.53s/it]INFO:__main__:Requesting http://fred-wang.github.io/MathUI2014/demos/7-web-component.html
INFO:__main__:Getting metadata for http://fred-wang.github.io/MathUI2014/demos/7-web-component.html
 40%|████      | 63/157 [01:18<01:46,  1.13s/it]INFO:__main__:Requesting https://github.com/fred-wang/MathUI2014/blob/master/demos/7-web-component.html
INFO:__main__:Getting metadata for https://github.com/fred-wang/MathUI2014/blob/master/demos/7-web-component.html
 41%|████      | 64/157 [01:19<01:32,  1.01it/s]INFO:__main__:Requesting https://github.com/fred-wang/x-tex
INFO:__main__:Getting metadata for https://github.com/fred-wang/x-tex
 41%|████▏     | 65/157 [01:20<01:23,  1.10it/s]INFO:__main__:Requesting https://runarberg.github.io/ascii2mathml/
INFO:__main__:Getting metadata for https://runarberg.github.io/ascii2mathml/
 42%|████▏     | 66/157 [01:20<01:16,  1.19it/s]INFO:__main__:Requesting http://docs.mathjax.org/en/latest/output.html
INFO:__main__:Getting metadata for http://docs.mathjax.org/en/latest/output.html
 43%|████▎     | 67/157 [01:21<01:17,  1.16it/s]INFO:__main__:Requesting https://webkit.org/blog/6803/improvements-in-mathml-rendering/
INFO:__main__:Getting metadata for https://webkit.org/blog/6803/improvements-in-mathml-rendering/
 43%|████▎     | 68/157 [01:22<01:09,  1.29it/s]INFO:__main__:Requesting https://standardebooks.org/ebooks/ludwig-wittgenstein/tractatus-logico-philosophicus/c-k-ogden
INFO:__main__:Getting metadata for https://standardebooks.org/ebooks/ludwig-wittgenstein/tractatus-logico-philosophicus/c-k-ogden
 44%|████▍     | 69/157 [01:22<00:57,  1.53it/s]INFO:__main__:Requesting https://en.m.wikipedia.org/wiki/Help:Displaying_a_formula#Native_MathML
INFO:__main__:Getting metadata for https://en.m.wikipedia.org/wiki/Help:Displaying_a_formula#Native_MathML
 45%|████▍     | 70/157 [01:29<03:29,  2.40s/it]INFO:__main__:Requesting http://www.w3.org/1998/Math/MathML">
 45%|████▌     | 71/157 [01:29<02:27,  1.72s/it]INFO:__main__:Requesting http://fred-wang.github.io/MathUI2014/demos/2-mathml-in-svg.svg
INFO:__main__:Getting metadata for http://fred-wang.github.io/MathUI2014/demos/2-mathml-in-svg.svg
 46%|████▌     | 72/157 [01:29<01:48,  1.28s/it]INFO:__main__:Requesting http://fred-wang.github.io/MathUI2014/demos/6-mathml-in-webgl.html
INFO:__main__:Getting metadata for http://fred-wang.github.io/MathUI2014/demos/6-mathml-in-webgl.html
 46%|████▋     | 73/157 [01:29<01:19,  1.06it/s]INFO:__main__:Requesting https://www.mediawiki.org/wiki/Extension:Math/advancedSettings#CSS_for_the_MathML_with_SVG_fallback_mode
INFO:__main__:Getting metadata for https://www.mediawiki.org/wiki/Extension:Math/advancedSettings#CSS_for_the_MathML_with_SVG_fallback_mode
 47%|████▋     | 74/157 [01:30<01:20,  1.04it/s]INFO:__main__:Requesting https://ncatlab.org/nlab/show/HomePage
INFO:__main__:Getting metadata for https://ncatlab.org/nlab/show/HomePage
 48%|████▊     | 75/157 [01:32<01:28,  1.08s/it]INFO:__main__:Requesting https://arxiv.org/help/mathjax
INFO:__main__:Getting metadata for https://arxiv.org/help/mathjax
 48%|████▊     | 76/157 [01:34<01:48,  1.34s/it]INFO:__main__:Requesting https://mdn.mozillademos.org/en-US/docs/Mozilla/MathML_Project/MathML_Torture_Test$samples/MathML_Torture_Test?revision=1449367
INFO:__main__:Getting metadata for https://mdn.mozillademos.org/en-US/docs/Mozilla/MathML_Project/MathML_Torture_Test$samples/MathML_Torture_Test?revision=1449367
 49%|████▉     | 77/157 [01:35<01:58,  1.48s/it]INFO:__main__:Requesting https://bugs.chromium.org/p/chromium/issues/list?can=2&q=%22use+after+free%22&sort=&groupby=&colspec=&x=&y=&mode=&cells=&num=
INFO:__main__:Getting metadata for https://bugs.chromium.org/p/chromium/issues/list?can=2&q=%22use+after+free%22&sort=&groupby=&colspec=&x=&y=&mode=&cells=&num=
 50%|████▉     | 78/157 [01:43<04:23,  3.34s/it]INFO:__main__:Requesting https://www.gwern.net/Embryo-selection
INFO:__main__:Getting metadata for https://www.gwern.net/Embryo-selection
 50%|█████     | 79/157 [01:49<05:10,  3.99s/it]INFO:__main__:Requesting https://github.com/Microsoft/reactxp
INFO:__main__:Getting metadata for https://github.com/Microsoft/reactxp
 51%|█████     | 80/157 [01:49<03:55,  3.05s/it]INFO:__main__:Requesting https://twitter.com/joeerl/status/1104298407231922176
INFO:__main__:Getting metadata for https://twitter.com/joeerl/status/1104298407231922176
 52%|█████▏    | 81/157 [01:51<03:10,  2.51s/it]INFO:__main__:Requesting http://paulgraham.com/pypar.html
INFO:__main__:Getting metadata for http://paulgraham.com/pypar.html
 52%|█████▏    | 82/157 [01:52<02:48,  2.25s/it]INFO:__main__:Requesting https://elixir-lang.org/getting-started/introduction.html
INFO:__main__:Getting metadata for https://elixir-lang.org/getting-started/introduction.html
 53%|█████▎    | 83/157 [01:53<02:01,  1.64s/it]INFO:__main__:Requesting https://thedailywtf.com/articles/A_Case_of_the_MUMPS
INFO:__main__:Getting metadata for https://thedailywtf.com/articles/A_Case_of_the_MUMPS
 54%|█████▎    | 84/157 [01:55<02:11,  1.81s/it]INFO:__main__:Requesting https://twitter.com/devoncestes/status/1104000439987683328
INFO:__main__:Getting metadata for https://twitter.com/devoncestes/status/1104000439987683328
 54%|█████▍    | 85/157 [01:56<01:48,  1.51s/it]INFO:__main__:Requesting http://paulgraham.com/avg.html
INFO:__main__:Getting metadata for http://paulgraham.com/avg.html
 55%|█████▍    | 86/157 [01:57<01:53,  1.59s/it]INFO:__main__:Requesting https://www.activestate.com/resources/webinars/perl-paradox/
INFO:__main__:Getting metadata for https://www.activestate.com/resources/webinars/perl-paradox/
 55%|█████▌    | 87/157 [01:58<01:30,  1.29s/it]INFO:__main__:Requesting https://learnyousomeerlang.com/dialyzer
INFO:__main__:Getting metadata for https://learnyousomeerlang.com/dialyzer
 56%|█████▌    | 88/157 [02:00<01:39,  1.44s/it]INFO:__main__:Requesting http://wiki.c2.com/?SoftTyping
INFO:__main__:Getting metadata for http://wiki.c2.com/?SoftTyping
 57%|█████▋    | 89/157 [02:00<01:13,  1.08s/it]INFO:__main__:Requesting https://github.com/patriknw/akka-typed-blog/blob/master/src/main/java/blog/typed/javadsl/Greeter1.java
INFO:__main__:Getting metadata for https://github.com/patriknw/akka-typed-blog/blob/master/src/main/java/blog/typed/javadsl/Greeter1.java
 57%|█████▋    | 90/157 [02:01<01:05,  1.03it/s]INFO:__main__:Requesting https://www.theparisreview.org/blog/2019/03/05/dice-roll-the-phantom-gambler/
INFO:__main__:Getting metadata for https://www.theparisreview.org/blog/2019/03/05/dice-roll-the-phantom-gambler/
 58%|█████▊    | 91/157 [02:01<00:49,  1.32it/s]INFO:__main__:Requesting https://www.workable.com/j/089F60DE31
INFO:__main__:Getting metadata for https://jerry.workable.com/j/089F60DE31
 59%|█████▊    | 92/157 [02:02<00:50,  1.30it/s]INFO:__main__:Requesting https://github.com/DeanRoddey/CIDLib#cidlib---a-c-development-environment
INFO:__main__:Getting metadata for https://github.com/DeanRoddey/CIDLib#cidlib---a-c-development-environment
 59%|█████▉    | 93/157 [02:02<00:49,  1.29it/s]INFO:__main__:Requesting https://www.jefftk.com/p/nomic-game-3
INFO:__main__:Getting metadata for https://www.jefftk.com/p/nomic-game-3
 60%|█████▉    | 94/157 [02:05<01:17,  1.22s/it]INFO:__main__:Requesting https://agoranomic.org
INFO:__main__:Getting metadata for https://agoranomic.org
 61%|██████    | 95/157 [02:06<01:08,  1.11s/it]INFO:__main__:Requesting https://www.bu.edu/research/articles/researchers-develop-acoustic-metamaterial-noise-cancellation-device/
INFO:__main__:Getting metadata for https://www.bu.edu/research/articles/researchers-develop-acoustic-metamaterial-noise-cancellation-device/
 61%|██████    | 96/157 [02:06<01:01,  1.01s/it]INFO:__main__:Requesting https://www.acousticfields.com/how-do-you-stop-low-frequency-issues-in-a-small-room/
INFO:__main__:Getting metadata for https://www.acousticfields.com/how-do-you-stop-low-frequency-issues-in-a-small-room/
 62%|██████▏   | 97/157 [02:07<00:48,  1.23it/s]INFO:__main__:Requesting https://www.youtube.com/watch?v=Mv6sBuwzLhk
INFO:__main__:Getting metadata for https://www.youtube.com/watch?v=Mv6sBuwzLhk
 62%|██████▏   | 98/157 [02:08<00:49,  1.19it/s]INFO:__main__:Requesting https://en.wikipedia.org/wiki/Fletcher%E2%80%93Munson_curves
INFO:__main__:Getting metadata for https://en.wikipedia.org/wiki/Fletcher%E2%80%93Munson_curves
 63%|██████▎   | 99/157 [02:08<00:47,  1.21it/s]INFO:__main__:Requesting https://journals.aps.org/prb/abstract/10.1103/PhysRevB.99.024302
INFO:__main__:Getting metadata for https://journals.aps.org/prb/abstract/10.1103/PhysRevB.99.024302
 64%|██████▎   | 100/157 [02:09<00:46,  1.23it/s]INFO:__main__:Requesting https://www.insulationsuperstore.co.uk/product/soundboard-3-acoustic-wall-board-46db-reduction-1-2m-x-1m-x-24mm.html
INFO:__main__:Getting metadata for https://www.insulationsuperstore.co.uk/product/soundboard-3-acoustic-wall-board-46db-reduction-1-2m-x-1m-x-24mm.html
 64%|██████▍   | 101/157 [02:11<01:07,  1.21s/it]INFO:__main__:Requesting https://www.bu.edu/research/files/2019/02/acoustic-metamaterial-noise-cancellation-device-19-1023-SOUND-060-2.jpg
INFO:__main__:Getting metadata for https://www.bu.edu/research/files/2019/02/acoustic-metamaterial-noise-cancellation-device-19-1023-SOUND-060-2.jpg
 65%|██████▍   | 102/157 [02:15<01:49,  1.99s/it]INFO:__main__:Requesting https://simple.wikipedia.org/wiki/Decibel
INFO:__main__:Getting metadata for https://simple.wikipedia.org/wiki/Decibel
 66%|██████▌   | 103/157 [02:16<01:24,  1.57s/it]INFO:__main__:Requesting https://en.wikipedia.org/wiki/Decibel
INFO:__main__:Getting metadata for https://en.wikipedia.org/wiki/Decibel
 66%|██████▌   | 104/157 [02:18<01:31,  1.73s/it]INFO:__main__:Requesting https://www.amazon.com/dp/B004U4A5RU/
INFO:__main__:Getting metadata for https://www.amazon.com/dp/B004U4A5RU/
 67%|██████▋   | 105/157 [02:21<01:48,  2.10s/it]INFO:__main__:Requesting https://www.amazon.com/dp/B00RM6Q9XW/
INFO:__main__:Getting metadata for https://www.amazon.com/dp/B00RM6Q9XW/
 68%|██████▊   | 106/157 [02:24<02:04,  2.44s/it]INFO:__main__:Requesting https://www.amazon.com/dp/B0015PN3W6/
INFO:__main__:Getting metadata for https://www.amazon.com/dp/B0015PN3W6/
 68%|██████▊   | 107/157 [02:27<02:05,  2.52s/it]INFO:__main__:Requesting https://www.youtube.com/watch?v=5qV5j9wD5e8
INFO:__main__:Getting metadata for https://www.youtube.com/watch?v=5qV5j9wD5e8
 69%|██████▉   | 108/157 [02:28<01:39,  2.04s/it]INFO:__main__:Requesting https://www.youtube.com/watch?v=TRKB5kWs7KE
INFO:__main__:Getting metadata for https://www.youtube.com/watch?v=TRKB5kWs7KE
 69%|██████▉   | 109/157 [02:29<01:21,  1.69s/it]INFO:__main__:Requesting https://journals.aps.org/prb/pdf/10.1103/PhysRevB.99.024302
 70%|███████   | 110/157 [02:29<00:59,  1.26s/it]INFO:__main__:Requesting https://medium.com/@niant/why-deadlines-and-sprints-are-bad-for-you-7ee87be5d0f0
INFO:__main__:Getting metadata for https://medium.com/@niant/why-deadlines-and-sprints-are-bad-for-you-7ee87be5d0f0
 71%|███████   | 111/157 [02:29<00:46,  1.02s/it]INFO:__main__:Requesting https://www.bbc.com/news/science-environment-47478537
INFO:__main__:Getting metadata for https://www.bbc.com/news/science-environment-47478537
 71%|███████▏  | 112/157 [02:29<00:35,  1.28it/s]INFO:__main__:Requesting https://www.theguardian.com/science/2005/jan/18/educationsgendergap.genderissues
INFO:__main__:Getting metadata for https://www.theguardian.com/science/2005/jan/18/educationsgendergap.genderissues
 72%|███████▏  | 113/157 [02:30<00:29,  1.52it/s]INFO:__main__:Requesting https://press.cern/press-releases/2018/09/updated-statement-cern-stands-diversity*
 73%|███████▎  | 114/157 [02:32<00:44,  1.03s/it]INFO:__main__:Requesting https://news.ycombinator.com/newsguidelines.html
INFO:__main__:Getting metadata for https://news.ycombinator.com/newsguidelines.html
 73%|███████▎  | 115/157 [02:33<00:44,  1.07s/it]INFO:__main__:Requesting https://drive.google.com/file/d/1c_NyUhOZ8erdqU2AGZJZtNfFeA91Kefj/view
INFO:__main__:Getting metadata for https://drive.google.com/file/d/1c_NyUhOZ8erdqU2AGZJZtNfFeA91Kefj/view
 74%|███████▍  | 116/157 [02:35<00:51,  1.26s/it]INFO:__main__:Requesting https://www.scottaaronson.com/blog/?p=304
ERROR:__main__:Could not reach https://www.scottaaronson.com/blog/?p=304
Traceback (most recent call last):
  File "/home/travis/virtualenv/python3.6.3/lib/python3.6/site-packages/urllib3/connectionpool.py", line 384, in _make_request
    six.raise_from(e, None)
  File "<string>", line 2, in raise_from
  File "/home/travis/virtualenv/python3.6.3/lib/python3.6/site-packages/urllib3/connectionpool.py", line 380, in _make_request
    httplib_response = conn.getresponse()
  File "/opt/python/3.6.3/lib/python3.6/http/client.py", line 1331, in getresponse
    response.begin()
  File "/opt/python/3.6.3/lib/python3.6/http/client.py", line 297, in begin
    version, status, reason = self._read_status()
  File "/opt/python/3.6.3/lib/python3.6/http/client.py", line 258, in _read_status
    line = str(self.fp.readline(_MAXLINE + 1), "iso-8859-1")
  File "/opt/python/3.6.3/lib/python3.6/socket.py", line 586, in readinto
    return self._sock.recv_into(b)
  File "/opt/python/3.6.3/lib/python3.6/ssl.py", line 1009, in recv_into
    return self.read(nbytes, buffer)
  File "/opt/python/3.6.3/lib/python3.6/ssl.py", line 871, in read
    return self._sslobj.read(len, buffer)
  File "/opt/python/3.6.3/lib/python3.6/ssl.py", line 631, in read
    v = self._sslobj.read(len, buffer)
socket.timeout: The read operation timed out

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/travis/virtualenv/python3.6.3/lib/python3.6/site-packages/requests/adapters.py", line 449, in send
    timeout=timeout
  File "/home/travis/virtualenv/python3.6.3/lib/python3.6/site-packages/urllib3/connectionpool.py", line 638, in urlopen
    _stacktrace=sys.exc_info()[2])
  File "/home/travis/virtualenv/python3.6.3/lib/python3.6/site-packages/urllib3/util/retry.py", line 367, in increment
    raise six.reraise(type(error), error, _stacktrace)
  File "/home/travis/virtualenv/python3.6.3/lib/python3.6/site-packages/urllib3/packages/six.py", line 686, in reraise
    raise value
  File "/home/travis/virtualenv/python3.6.3/lib/python3.6/site-packages/urllib3/connectionpool.py", line 600, in urlopen
    chunked=chunked)
  File "/home/travis/virtualenv/python3.6.3/lib/python3.6/site-packages/urllib3/connectionpool.py", line 386, in _make_request
    self._raise_timeout(err=e, url=url, timeout_value=read_timeout)
  File "/home/travis/virtualenv/python3.6.3/lib/python3.6/site-packages/urllib3/connectionpool.py", line 306, in _raise_timeout
    raise ReadTimeoutError(self, url, "Read timed out. (read timeout=%s)" % timeout_value)
urllib3.exceptions.ReadTimeoutError: HTTPSConnectionPool(host='www.scottaaronson.com', port=443): Read timed out. (read timeout=6)

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "../_scripts/crawler/crawler.py", line 210, in metafy_url
    response = requests.get(url, timeout=6, headers={'User-Agent': 'Mozilla/5.0'})
  File "/home/travis/virtualenv/python3.6.3/lib/python3.6/site-packages/requests/api.py", line 75, in get
    return request('get', url, params=params, **kwargs)
  File "/home/travis/virtualenv/python3.6.3/lib/python3.6/site-packages/requests/api.py", line 60, in request
    return session.request(method=method, url=url, **kwargs)
  File "/home/travis/virtualenv/python3.6.3/lib/python3.6/site-packages/requests/sessions.py", line 533, in request
    resp = self.send(prep, **send_kwargs)
  File "/home/travis/virtualenv/python3.6.3/lib/python3.6/site-packages/requests/sessions.py", line 646, in send
    r = adapter.send(request, **kwargs)
  File "/home/travis/virtualenv/python3.6.3/lib/python3.6/site-packages/requests/adapters.py", line 529, in send
    raise ReadTimeout(e, request=request)
requests.exceptions.ReadTimeout: HTTPSConnectionPool(host='www.scottaaronson.com', port=443): Read timed out. (read timeout=6)
 75%|███████▍  | 117/157 [02:41<01:51,  2.79s/it]INFO:__main__:Requesting http://resonaances.blogspot.com/2016/06/game-of-thrones-750-gev-edition.html
INFO:__main__:Getting metadata for http://resonaances.blogspot.com/2016/06/game-of-thrones-750-gev-edition.html
 75%|███████▌  | 118/157 [02:41<01:21,  2.09s/it]INFO:__main__:Requesting https://www.particlesforjustice.org
INFO:__main__:Getting metadata for https://www.particlesforjustice.org
 76%|███████▌  | 119/157 [02:42<01:04,  1.70s/it]INFO:__main__:Requesting https://justiceforstrumia.org
INFO:__main__:Getting metadata for https://justiceforstrumia.org
 76%|███████▋  | 120/157 [02:43<00:52,  1.41s/it]INFO:__main__:Requesting http://backreaction.blogspot.com/2018/10/gender-bias-in-academia-case-strumia.html?m=1
INFO:__main__:Getting metadata for http://backreaction.blogspot.com/2018/10/gender-bias-in-academia-case-strumia.html?m=1
 77%|███████▋  | 121/157 [02:44<00:44,  1.23s/it]INFO:__main__:Requesting https://en.m.wikipedia.org/wiki/Emmy_Noether
INFO:__main__:Getting metadata for https://en.m.wikipedia.org/wiki/Emmy_Noether
 78%|███████▊  | 122/157 [02:46<00:50,  1.43s/it]INFO:__main__:Requesting https://en.wikipedia.org/wiki/The_Emperor%27s_New_Clothes
INFO:__main__:Getting metadata for https://en.wikipedia.org/wiki/The_Emperor%27s_New_Clothes
 78%|███████▊  | 123/157 [02:47<00:46,  1.36s/it]INFO:__main__:Requesting http://gltr.io/
INFO:__main__:Getting metadata for http://gltr.io
 79%|███████▉  | 124/157 [02:47<00:33,  1.00s/it]INFO:__main__:Requesting https://en.wikipedia.org/wiki/Mojibake
INFO:__main__:Getting metadata for https://en.wikipedia.org/wiki/Mojibake
 80%|███████▉  | 125/157 [02:49<00:40,  1.27s/it]INFO:__main__:Requesting https://dwheeler.com/essays/fixing-unix-linux-filenames.html
INFO:__main__:Getting metadata for https://dwheeler.com/essays/fixing-unix-linux-filenames.html
 80%|████████  | 126/157 [02:50<00:41,  1.33s/it]INFO:__main__:Requesting https://lobste.rs/s/luehpr/fixing_unix_linux_posix_filenames
INFO:__main__:Getting metadata for https://lobste.rs/s/luehpr/fixing_unix_linux_posix_filenames
 81%|████████  | 127/157 [02:51<00:36,  1.21s/it]INFO:__main__:Requesting https://github.com/pwaller/go2ll-talk
INFO:__main__:Getting metadata for https://github.com/pwaller/go2ll-talk
 82%|████████▏ | 128/157 [02:52<00:32,  1.12s/it]INFO:__main__:Requesting https://github.com/pwaller/go2ll-talk/blob/master/Makefile
INFO:__main__:Getting metadata for https://github.com/pwaller/go2ll-talk/blob/master/Makefile
 82%|████████▏ | 129/157 [02:53<00:27,  1.03it/s]INFO:__main__:Requesting https://play.golang.org/p/3dVFCT2-kN8
INFO:__main__:Getting metadata for https://play.golang.org/p/3dVFCT2-kN8
 83%|████████▎ | 130/157 [02:53<00:22,  1.22it/s]INFO:__main__:Requesting https://golang.org/src/go/constant/value.go?#L58
INFO:__main__:Getting metadata for https://golang.org/src/go/constant/value.go#L58
 83%|████████▎ | 131/157 [02:54<00:21,  1.20it/s]INFO:__main__:Requesting https://www.mercurynews.com/2019/03/08/h-1b-outsourcing-giant-games-visa-system-to-discriminate-against-non-south-asians-in-hiring-lawsuit-claims/
INFO:__main__:Getting metadata for https://www.mercurynews.com/2019/03/08/h-1b-outsourcing-giant-games-visa-system-to-discriminate-against-non-south-asians-in-hiring-lawsuit-claims/
 84%|████████▍ | 132/157 [02:55<00:17,  1.43it/s]INFO:__main__:Requesting https://en.wikipedia.org/wiki/Reverse_auction
INFO:__main__:Getting metadata for https://en.wikipedia.org/wiki/Reverse_auction
 85%|████████▍ | 133/157 [02:55<00:18,  1.30it/s]INFO:__main__:Requesting https://github.com/serv/h1bhub
INFO:__main__:Getting metadata for https://github.com/serv/h1bhub
 85%|████████▌ | 134/157 [02:56<00:19,  1.20it/s]INFO:__main__:Requesting https://m.economictimes.com/nri/visa-and-immigration/h-1b-extension-rejections-rob-indian-it-firms-of-visa-power/articleshow/68312136.cms
INFO:__main__:Getting metadata for https://economictimes.indiatimes.com/nri/visa-and-immigration/h-1b-extension-rejections-rob-indian-it-firms-of-visa-power/articleshow/68312136.cms?from=mdr
 86%|████████▌ | 135/157 [02:58<00:21,  1.04it/s]INFO:__main__:Requesting https://m.timesofindia.com/business/india-business/tcs-among-top-10-firms-to-get-foreign-labour-certification-for-h-1b-visas/articleshow/66328540.cms
INFO:__main__:Getting metadata for https://timesofindia.indiatimes.com/business/india-business/tcs-among-top-10-firms-to-get-foreign-labour-certification-for-h-1b-visas/articleshow/66328540.cms?from=mdr
 87%|████████▋ | 136/157 [02:59<00:22,  1.06s/it]INFO:__main__:Requesting https://www.engadget.com/2019/03/09/porche-taycan-ev-20-000-reservations/
INFO:__main__:Getting metadata for https://www.engadget.com/2019/03/09/porche-taycan-ev-20-000-reservations/
 87%|████████▋ | 137/157 [03:00<00:19,  1.01it/s]INFO:__main__:Requesting https://my.porsche.com/germany/microsite/e-performance-deposit/
INFO:__main__:Getting metadata for https://my.porsche.com/germany/microsite/e-performance-deposit/
 88%|████████▊ | 138/157 [03:01<00:21,  1.14s/it]INFO:__main__:Requesting https://www.polestar.com/
INFO:__main__:Getting metadata for https://www.polestar.com
 89%|████████▊ | 139/157 [03:03<00:25,  1.42s/it]INFO:__main__:Requesting https://earlymodernmedicine.com/the-man-with-an-elephants-nose/
 89%|████████▉ | 140/157 [03:04<00:19,  1.13s/it]INFO:__main__:Requesting https://youtu.be/WaVzajICS-w
INFO:__main__:Getting metadata for https://www.youtube.com/watch?v=WaVzajICS-w&feature=youtu.be
 90%|████████▉ | 141/157 [03:05<00:16,  1.04s/it]INFO:__main__:Requesting https://en.wikipedia.org/wiki/Alejandro_Jodorowsky#Psychomagic
INFO:__main__:Getting metadata for https://en.wikipedia.org/wiki/Alejandro_Jodorowsky#Psychomagic
 90%|█████████ | 142/157 [03:07<00:19,  1.28s/it]INFO:__main__:Requesting https://www.sciencemag.org/news/2019/03/black-death-may-have-transformed-medieval-societies-sub-saharan-africa
INFO:__main__:Getting metadata for https://www.sciencemag.org/news/2019/03/black-death-may-have-transformed-medieval-societies-sub-saharan-africa
 91%|█████████ | 143/157 [03:07<00:14,  1.06s/it]INFO:__main__:Requesting http://nautil.us/issue/70/variables/cloudy-with-a-chance-of-war-rp
INFO:__main__:Getting metadata for http://nautil.us/issue/70/variables/cloudy-with-a-chance-of-war-rp
 92%|█████████▏| 144/157 [03:09<00:18,  1.39s/it]INFO:__main__:Requesting https://www.theatlantic.com/technology/archive/2019/03/ai-created-art-invades-chelsea-gallery-scene/584134/
INFO:__main__:Getting metadata for https://www.theatlantic.com/technology/archive/2019/03/ai-created-art-invades-chelsea-gallery-scene/584134/
 92%|█████████▏| 145/157 [03:09<00:12,  1.05s/it]INFO:__main__:Requesting https://thispersondoesnotexist.com/
INFO:__main__:Getting metadata for https://thispersondoesnotexist.com
ERROR:__main__:Could not get metadata for https://thispersondoesnotexist.com
Traceback (most recent call last):
  File "/home/travis/virtualenv/python3.6.3/lib/python3.6/site-packages/urllib3/connectionpool.py", line 600, in urlopen
    chunked=chunked)
  File "/home/travis/virtualenv/python3.6.3/lib/python3.6/site-packages/urllib3/connectionpool.py", line 384, in _make_request
    six.raise_from(e, None)
  File "<string>", line 2, in raise_from
  File "/home/travis/virtualenv/python3.6.3/lib/python3.6/site-packages/urllib3/connectionpool.py", line 380, in _make_request
    httplib_response = conn.getresponse()
  File "/opt/python/3.6.3/lib/python3.6/http/client.py", line 1331, in getresponse
    response.begin()
  File "/opt/python/3.6.3/lib/python3.6/http/client.py", line 297, in begin
    version, status, reason = self._read_status()
  File "/opt/python/3.6.3/lib/python3.6/http/client.py", line 266, in _read_status
    raise RemoteDisconnected("Remote end closed connection without"
http.client.RemoteDisconnected: Remote end closed connection without response

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/travis/virtualenv/python3.6.3/lib/python3.6/site-packages/requests/adapters.py", line 449, in send
    timeout=timeout
  File "/home/travis/virtualenv/python3.6.3/lib/python3.6/site-packages/urllib3/connectionpool.py", line 638, in urlopen
    _stacktrace=sys.exc_info()[2])
  File "/home/travis/virtualenv/python3.6.3/lib/python3.6/site-packages/urllib3/util/retry.py", line 367, in increment
    raise six.reraise(type(error), error, _stacktrace)
  File "/home/travis/virtualenv/python3.6.3/lib/python3.6/site-packages/urllib3/packages/six.py", line 685, in reraise
    raise value.with_traceback(tb)
  File "/home/travis/virtualenv/python3.6.3/lib/python3.6/site-packages/urllib3/connectionpool.py", line 600, in urlopen
    chunked=chunked)
  File "/home/travis/virtualenv/python3.6.3/lib/python3.6/site-packages/urllib3/connectionpool.py", line 384, in _make_request
    six.raise_from(e, None)
  File "<string>", line 2, in raise_from
  File "/home/travis/virtualenv/python3.6.3/lib/python3.6/site-packages/urllib3/connectionpool.py", line 380, in _make_request
    httplib_response = conn.getresponse()
  File "/opt/python/3.6.3/lib/python3.6/http/client.py", line 1331, in getresponse
    response.begin()
  File "/opt/python/3.6.3/lib/python3.6/http/client.py", line 297, in begin
    version, status, reason = self._read_status()
  File "/opt/python/3.6.3/lib/python3.6/http/client.py", line 266, in _read_status
    raise RemoteDisconnected("Remote end closed connection without"
urllib3.exceptions.ProtocolError: ('Connection aborted.', RemoteDisconnected('Remote end closed connection without response',))

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/travis/virtualenv/python3.6.3/lib/python3.6/site-packages/webpreview/previews.py", line 34, in __init__
    res = requests.get(url, timeout=timeout, headers=headers)
  File "/home/travis/virtualenv/python3.6.3/lib/python3.6/site-packages/requests/api.py", line 75, in get
    return request('get', url, params=params, **kwargs)
  File "/home/travis/virtualenv/python3.6.3/lib/python3.6/site-packages/requests/api.py", line 60, in request
    return session.request(method=method, url=url, **kwargs)
  File "/home/travis/virtualenv/python3.6.3/lib/python3.6/site-packages/requests/sessions.py", line 533, in request
    resp = self.send(prep, **send_kwargs)
  File "/home/travis/virtualenv/python3.6.3/lib/python3.6/site-packages/requests/sessions.py", line 646, in send
    r = adapter.send(request, **kwargs)
  File "/home/travis/virtualenv/python3.6.3/lib/python3.6/site-packages/requests/adapters.py", line 498, in send
    raise ConnectionError(err, request=request)
requests.exceptions.ConnectionError: ('Connection aborted.', RemoteDisconnected('Remote end closed connection without response',))

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "../_scripts/crawler/crawler.py", line 93, in _get_site_title_and_description
    title, description, _ = web_preview(url, content=content)
  File "/home/travis/virtualenv/python3.6.3/lib/python3.6/site-packages/webpreview/previews.py", line 190, in web_preview
    og = OpenGraph(url, ['og:title', 'og:description', 'og:image'], timeout=timeout, headers=headers, content=content)
  File "/home/travis/virtualenv/python3.6.3/lib/python3.6/site-packages/webpreview/previews.py", line 165, in __init__
    super(OpenGraph, self).__init__(*args, **kwargs)
  File "/home/travis/virtualenv/python3.6.3/lib/python3.6/site-packages/webpreview/previews.py", line 131, in __init__
    super(SocialPreviewBase, self).__init__(*args, **kwargs)
  File "/home/travis/virtualenv/python3.6.3/lib/python3.6/site-packages/webpreview/previews.py", line 36, in __init__
    raise URLUnreachable("The URL does not exist.")
webpreview.excepts.URLUnreachable: The URL does not exist.
 93%|█████████▎| 146/157 [03:11<00:12,  1.11s/it]INFO:__main__:Requesting http://donellameadows.org/archives/leverage-points-places-to-intervene-in-a-system/
INFO:__main__:Getting metadata for http://donellameadows.org/archives/leverage-points-places-to-intervene-in-a-system/
 94%|█████████▎| 147/157 [03:12<00:10,  1.05s/it]INFO:__main__:Requesting https://www.francis-bacon.com/artworks/paintings/1960s
INFO:__main__:Getting metadata for https://www.francis-bacon.com/artworks/paintings/1960s
 94%|█████████▍| 148/157 [03:16<00:17,  2.00s/it]INFO:__main__:Requesting https://i.imgur.com/VeLbX27.jpg
INFO:__main__:Getting metadata for https://i.imgur.com/VeLbX27.jpg
 95%|█████████▍| 149/157 [03:22<00:26,  3.28s/it]INFO:__main__:Requesting https://en.wikipedia.org/wiki/Fountain_(Duchamp)
INFO:__main__:Getting metadata for https://en.wikipedia.org/wiki/Fountain_(Duchamp)
 96%|█████████▌| 150/157 [03:24<00:19,  2.77s/it]INFO:__main__:Requesting https://blog.acolyer.org/2019/03/08/a-generalised-solution-to-distributed-consensus/
ERROR:__main__:Could not reach https://blog.acolyer.org/2019/03/08/a-generalised-solution-to-distributed-consensus/
Traceback (most recent call last):
  File "/home/travis/virtualenv/python3.6.3/lib/python3.6/site-packages/urllib3/connectionpool.py", line 600, in urlopen
    chunked=chunked)
  File "/home/travis/virtualenv/python3.6.3/lib/python3.6/site-packages/urllib3/connectionpool.py", line 384, in _make_request
    six.raise_from(e, None)
  File "<string>", line 2, in raise_from
  File "/home/travis/virtualenv/python3.6.3/lib/python3.6/site-packages/urllib3/connectionpool.py", line 380, in _make_request
    httplib_response = conn.getresponse()
  File "/opt/python/3.6.3/lib/python3.6/http/client.py", line 1331, in getresponse
    response.begin()
  File "/opt/python/3.6.3/lib/python3.6/http/client.py", line 297, in begin
    version, status, reason = self._read_status()
  File "/opt/python/3.6.3/lib/python3.6/http/client.py", line 266, in _read_status
    raise RemoteDisconnected("Remote end closed connection without"
http.client.RemoteDisconnected: Remote end closed connection without response

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/travis/virtualenv/python3.6.3/lib/python3.6/site-packages/requests/adapters.py", line 449, in send
    timeout=timeout
  File "/home/travis/virtualenv/python3.6.3/lib/python3.6/site-packages/urllib3/connectionpool.py", line 638, in urlopen
    _stacktrace=sys.exc_info()[2])
  File "/home/travis/virtualenv/python3.6.3/lib/python3.6/site-packages/urllib3/util/retry.py", line 367, in increment
    raise six.reraise(type(error), error, _stacktrace)
  File "/home/travis/virtualenv/python3.6.3/lib/python3.6/site-packages/urllib3/packages/six.py", line 685, in reraise
    raise value.with_traceback(tb)
  File "/home/travis/virtualenv/python3.6.3/lib/python3.6/site-packages/urllib3/connectionpool.py", line 600, in urlopen
    chunked=chunked)
  File "/home/travis/virtualenv/python3.6.3/lib/python3.6/site-packages/urllib3/connectionpool.py", line 384, in _make_request
    six.raise_from(e, None)
  File "<string>", line 2, in raise_from
  File "/home/travis/virtualenv/python3.6.3/lib/python3.6/site-packages/urllib3/connectionpool.py", line 380, in _make_request
    httplib_response = conn.getresponse()
  File "/opt/python/3.6.3/lib/python3.6/http/client.py", line 1331, in getresponse
    response.begin()
  File "/opt/python/3.6.3/lib/python3.6/http/client.py", line 297, in begin
    version, status, reason = self._read_status()
  File "/opt/python/3.6.3/lib/python3.6/http/client.py", line 266, in _read_status
    raise RemoteDisconnected("Remote end closed connection without"
urllib3.exceptions.ProtocolError: ('Connection aborted.', RemoteDisconnected('Remote end closed connection without response',))

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "../_scripts/crawler/crawler.py", line 210, in metafy_url
    response = requests.get(url, timeout=6, headers={'User-Agent': 'Mozilla/5.0'})
  File "/home/travis/virtualenv/python3.6.3/lib/python3.6/site-packages/requests/api.py", line 75, in get
    return request('get', url, params=params, **kwargs)
  File "/home/travis/virtualenv/python3.6.3/lib/python3.6/site-packages/requests/api.py", line 60, in request
    return session.request(method=method, url=url, **kwargs)
  File "/home/travis/virtualenv/python3.6.3/lib/python3.6/site-packages/requests/sessions.py", line 533, in request
    resp = self.send(prep, **send_kwargs)
  File "/home/travis/virtualenv/python3.6.3/lib/python3.6/site-packages/requests/sessions.py", line 646, in send
    r = adapter.send(request, **kwargs)
  File "/home/travis/virtualenv/python3.6.3/lib/python3.6/site-packages/requests/adapters.py", line 498, in send
    raise ConnectionError(err, request=request)
requests.exceptions.ConnectionError: ('Connection aborted.', RemoteDisconnected('Remote end closed connection without response',))
 96%|█████████▌| 151/157 [03:24<00:11,  1.99s/it]INFO:__main__:Requesting http://hh360.user.srcf.net/blog/2016/08/majority-agreement-is-not-necessary/
INFO:__main__:Getting metadata for http://hh360.user.srcf.net/blog/2016/08/majority-agreement-is-not-necessary/
 97%|█████████▋| 152/157 [03:26<00:10,  2.08s/it]INFO:__main__:Requesting https://youtu.be/gYkueS5sKqo
INFO:__main__:Getting metadata for https://www.youtube.com/watch?v=gYkueS5sKqo&feature=youtu.be
 97%|█████████▋| 153/157 [03:27<00:06,  1.72s/it]INFO:__main__:Requesting https://www.brianstorti.com/raft/
INFO:__main__:Getting metadata for https://www.brianstorti.com/raft/
 98%|█████████▊| 154/157 [03:28<00:04,  1.43s/it]INFO:__main__:Requesting https://vadosware.io/post/paxosmon-gotta-concensus-them-all/
INFO:__main__:Getting metadata for https://vadosware.io/post/paxosmon-gotta-concensus-them-all/
 99%|█████████▊| 155/157 [03:29<00:02,  1.44s/it]INFO:__main__:Requesting https://en.bitcoin.it/wiki/Block_chain
INFO:__main__:Getting metadata for https://en.bitcoin.it/wiki/Block_chain
 99%|█████████▉| 156/157 [03:30<00:01,  1.34s/it]INFO:__main__:Requesting https://en.m.wikipedia.org/wiki/Byzantine_fault
INFO:__main__:Getting metadata for https://en.m.wikipedia.org/wiki/Byzantine_fault
100%|██████████| 157/157 [03:31<00:00,  1.18s/it]
